test/testpulltask.py

```diff
@@ -194,6 +194,8 @@ def run(self, opts, github):
         if not ret:
             proc = subprocess.Popen(cmd)
             ret = testinfra.wait_testing(proc, lambda: self.check_publishing(github))
+            if ret == 124:
```

Comment:
> Yes, but that happens only when we explicitly use timeout -s SIGKILL, no?

You are correct. Comments in the source code (timeout.c of coreutils):
```
If user specifies the KILL (9) signal is to be sent on timeout,
     the monitor is killed and so exits with 128+9 rather than 124.
```